assets/js/base/components/product-list/style.scss

```diff
@@ -222,6 +199,70 @@
 	}
 }
 
+.editor-styles-wrapper .wc-block-grid__products .wc-block-grid__product .wc-block-grid__product-onsale,
```

Comment:
I think this one can also be removed.

Reply:
These are required for storefront since it uses more specific selectors.